content/_changelogs/9.3.0.md

```diff
@@ -0,0 +1,54 @@
+## 9.3.0
+
+_Released 1/18/2022_
+
+**News:**
+
+- We've
+  [written a blog post](https://cypress.io/blog/2022/01/18/uploading-files-with-selectfile/)
+  about the new Cypress command, `cy.selectFile()` introduced in `9.3.0`.
+
+**Features:**
+
+- Added a new Cypress command, called `cy.selectFile()`, which provides the
+  ability to submit a file to file input elements. This command can also
+  simulate a user drag-dropping files over the document. The API is similar to
+  `cypress-file-upload`, and we have provided a migration guide for previous
+  users of that plugin. For full details, see
+  [our blog post](https://cypress.io/blog/2022/01/18/uploading-files-with-selectfile/)
+  and the
+  [cy.selectFile() documentation](https://docs.cypress.io/api/commands/selectfile).
+  Fixes [#19524](https://github.com/cypress-io/cypress/issues/19524) and
+  [#170](https://github.com/cypress-io/cypress/issues/170).
```

Comment:
Let's match the wording from the selectFile doc itself.

Commands that are chained (child commands) do not have `cy.` prefixed.

Link to the migration guide where we mentioned the migration guide. 

```suggestion
- You can now selects a file or files in an HTML5 input element or simulates dragging a file
or files into the browser using `.selectFile()`. The API is similar to
  `cypress-file-upload` and we have provided a migration guide for previous
  users of that plugin. For full details, see
  [our blog post](https://cypress.io/blog/2022/01/18/uploading-files-with-selectfile/)
  and the
  [.selectFile() documentation](https://docs.cypress.io/api/commands/selectfile).
  Fixes [#19524](https://github.com/cypress-io/cypress/issues/19524) and
  [#170](https://github.com/cypress-io/cypress/issues/170).
```

Reply:
Also these docs links need to be updated to internal links.